### ISSUE TITLE
Finder integration does not work on macOS Monterey

### ISSUE THREAD

**Issue by rhymeswithmogul** [keybase/client#24686]
I cannot add the FUSE driver on macOS 12.  When I click on "Yes, enable" it spins but nothing happens.
<img width="868" alt="image" src="https://user-images.githubusercontent.com/19931245/141305930-3901394f-76ca-4592-8b14-f6be0e337427.png">

macOS version 12.0.1
Keybase version 5.8.0-20210920184218+ac2177aec7
my log id: 40bfec503c26ad53cd80d01c

**Comment by rhymeswithmogul** [comment]
Here's what happens when I run the installer manually:

```
PS /Users/colin> sudo /Applications/Keybase.app/Contents/Resources/KeybaseInstaller.app/Contents/MacOS/Keybase  --app-path=/Applications/Keybase.app --run-mode=prod --timeout=60 --install-fuse                                                
11.11.2021 08:28:48.548 KBHelperTool:73[INFO] Connecting to keybase.Helper (privileged=1)
11.11.2021 08:28:48.548 KBHelperTool:73[INFO] Connected
11.11.2021 08:28:48.558 KBHelperTool:144[INFO] Running Version: 1.0.47
11.11.2021 08:28:48.583 KBHelperTool:144[INFO] Running Version: 1.0.47
11.11.2021 08:28:48.647 KBFuseComponent:64[ERROR] Invalid data: 
11.11.2021 08:28:48.662 KBFuseComponent:168[INFO] Current install needs fix
11.11.2021 08:28:48.822 KBFuseComponent:64[ERROR] Invalid data: 
11.11.2021 08:28:48.889 KBFuseComponent:64[ERROR] Invalid data: 
11.11.2021 08:28:48.899 KBInstaller:45[INFO] Install complete
11.11.2021 08:28:48.899 KBInstaller:49[INFO] Privileged Helper: Installed, Bundle Version: 1.0.47
Version: 1.0.47
11.11.2021 08:28:48.899 KBInstaller:49[INFO] Fuse: Error: The data couldn’t be read because it isn’t in the correct format.
Error
11.11.2021 08:28:48.900 Installer:97[INFO] Exit(1)
```

**Comment by nilicule** [comment]
This seems to be a duplicate of #24636

**Comment by maxtaco** [comment]
Closed as duplicative of #24636
